docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -178,72 +188,93 @@
+          until: ("uploaded" in app_upload_status.stdout)
+          ignore_errors: yes
+
+        - name: Fail if deployment-manager app upload fails
+          fail:
+            msg: "The upload of deployment-manager system app has failed.
+                  Expected 'uploaded' state but the app is in
+                  '{{ app_upload_status.stdout }}' state "
+          when: not ("uploaded" in app_upload_status.stdout)
+
+        when: dm_app_status.stdout_lines|length == 0
 
-      - name: Upload deployment-manager app Helm Chart
-        shell: source /etc/platform/openrc; system application-upload {{ app_helm_chart.files[0].path }}
+      - name: Apply deployment-manager app
+        shell: source /etc/platform/openrc; system application-apply deployment-manager
+        when: dm_app_status.stdout != 'applied' and dm_app_status.stdout != 'applying'
 
-      - name: Wait deployment-manager app to become uploaded
+      - name: Wait deployment-manager app to become applied
         shell: source /etc/platform/openrc; system application-list --nowrap | grep deployment-manager | awk '{ print $10 }'
-        register: app_upload_status
-        retries: 10
-        delay: 6
-        until: ("uploaded" in app_upload_status.stdout)
+        register: app_apply_status
+        retries: 30
+        delay: 10
+        until: ("applied" in app_apply_status.stdout)
         ignore_errors: yes
 
-      - name: Fail if deployment-manager app upload fails
+      - name: Fail if deployment-manager app apply fails
         fail:
-          msg: "The upload of deployment-manager system app has failed.
-                Expected 'uploaded' state but the app is in
-                '{{ app_upload_status.stdout }}' state "
-        when: not ("uploaded" in app_upload_status.stdout)
-
-      when: dm_app_status.stdout_lines|length == 0
-
-    - name: Apply deployment-manager app
-      shell: source /etc/platform/openrc; system application-apply deployment-manager
-      when: dm_app_status.stdout != 'applied' and dm_app_status.stdout != 'applying'
-
-    - name: Wait deployment-manager app to become applied
-      shell: source /etc/platform/openrc; system application-list --nowrap | grep deployment-manager | awk '{ print $10 }'
-      register: app_apply_status
-      retries: 30
-      delay: 10
-      until: ("applied" in app_apply_status.stdout)
-      ignore_errors: yes
-
-    - name: Fail if deployment-manager app apply fails
-      fail:
-        msg: "The apply of DM system app has failed.
-              Expected 'applied' state but the app is in
-              '{{ app_apply_status.stdout }}' state."
-      when: not ("applied" in app_apply_status.stdout)
+          msg: "The apply of DM system app has failed.
+                Expected 'applied' state but the app is in
+                '{{ app_apply_status.stdout }}' state."
+        when: not ("applied" in app_apply_status.stdout)
+
+      when: get_software_version.stdout >= "24.09"
+
+    - block:
```

Comment:
In this block, please add the block to remove services and webhooks in: https://github.com/Wind-River/cloud-platform-deployment-manager/pull/326/files

Reply:
Added